crates/cast/src/lib.rs

```diff
@@ -238,56 +246,33 @@ pub async fn wait_for_tx(
     retries: u8,
 ) -> Result<&str> {
     println!("Transaction hash: {tx_hash:#x}");
-
-    let mut maybe_receipt: Option<MaybePendingTransactionReceipt> = None;
     for i in (1..retries).rev() {
```

Comment:
Didn't know that the famous `wait_for_tx` from starknet.py made it's apperance here in cast 😅

Reply:
Curtesy of Marcin :D